autoload/lsp/internal/work_done_progress.vim

```diff
@@ -0,0 +1,45 @@
+let s:lsp_progress = {
```

Comment:
Please add a link to  https://microsoft.github.io/language-server-protocol/specifications/specification-current/#progress at the top of the file as a comment.

Reply:
I fixed it.